### ISSUE TITLE
Optionally Remove Leading LF Before First "xmlns"

### ISSUE THREAD

**Issue by boozook** [DotJoshJohnson/vscode-xml#89]
Needed new feature.

`"xmlTools.splitFirstXmlnsOnFormat": false` - if `false` then first ns should not be aligned.

For example we have config:
```json
{
  "xmlTools.splitXmlnsOnFormat": true,
  "xmlTools.splitFirstXmlnsOnFormat": false
}
```

and xml:
```xml
<?xml version="1.0" encoding="utf-8" standalone="no"?>
<xml xmlns="http://www.w3.org/1999/xhtml" xmlns:soap="http://www.w3.org/2003/05/soap-envelope/" soap:encoding="http://www.w3.org/2003/05/soap-encoding">
	<node></node>
</xml>
```

So aligned xml should be that:

```xml
<?xml version="1.0" encoding="utf-8" standalone="no"?>
<xml xmlns="http://www.w3.org/1999/xhtml"
     xmlns:soap="http://www.w3.org/2003/05/soap-envelope/"
     soap:encoding="http://www.w3.org/2003/05/soap-encoding">
	<node></node>
</xml>
```

**Comment by DotJoshJohnson** [comment]
This will be the default behavior in the 2.x formatter (PR #141).

**Comment by DotJoshJohnson** [comment]
Released in [`v2.0.0-preview.1`](https://github.com/DotJoshJohnson/vscode-xml/releases/tag/v2.0.0-preview.1). This will be released to the marketplace when `v2.0.0` is out of preview.
